tests/BridgeImplementationTest.php

```diff
@@ -209,11 +209,15 @@ public function dataBridgesProvider() {
 		foreach (glob(PATH_LIB_BRIDGES . '*.php') as $path) {
 			$bridges[basename($path, '.php')] = array($path);
 		}
+
+		// Remove a few files that are not bridges
+		unset($bridges['PepperBridgeAbstract']);
+		unset($bridges['DanbooruBridge_Fix_Simple_Html_Dom']);
+
```

Comment:
Bridges must have a `Bridge` suffix in their file name, so instead of removing unwanted files manually, perhaps this can be solved by applying the correct filter.

```suggestion
		foreach (glob(PATH_LIB_BRIDGES . '*Bridge.php') as $path) {
			$bridges[basename($path, '.php')] = array($path);
		}

```